src/Neo/Cryptography/Hasher.cs

```diff
@@ -9,11 +9,14 @@
 // Redistribution and use in source and binary forms with or without
 // modifications are permitted.
 
+using System;
+
 namespace Neo.Cryptography
 {
     /// <summary>
     /// Represents hash function identifiers supported by ECDSA message signature and verification.
     /// </summary>
+    [Obsolete("Use HashAlgorithm instead")]
```

Comment:
I prefer rename the enum instead of duplicate all the methods

Reply:
You have to, those methods are public, previously our principle is duplicate a new one and obsolete old one before remove it, at least previouly we deal with public methods that way.